ios/Podfile

```diff
@@ -81,6 +81,17 @@ abstract_target 'Expo Go' do
         config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << 'ENABLE_PACKAGER_CONNECTION=0'
       end
     end
+
+    # Fix building failures on M1
+    installer.generated_projects.each do |project|
+      project.targets.each do |target|
+        target.build_configurations.each do |config|
+          config.build_settings["EXCLUDED_ARCHS[sdk=iphonesimulator*]"] = "arm64"
+        end
+      end
+      project.save()
```

Comment:
I see a small problem here — Xcode tries to reindex the entire project when its pbxproj file changes, so this means it may take more time after running `pod install` until you're able to build the project in Xcode 🤔
Can't we do this thing in `installer.target_installation_results.pod_target_installation_results`?

Reply:
yes, this works and updated.
didn't familiar enough for cocoapods, thanks for the great comment 👍